### ISSUE TITLE
FR: sorting on the package catalog is not clear or toggle-able

### ISSUE THREAD

**Issue by leeederek** [kurtosis-tech/kurtosis-package-indexer#27]
While on: https://cloud.kurtosis.com/enclave-manager, there's no easy way so sort the packages that are shown to me. I expected it to be alphabetically ordered and to have the ability to toggle how its sorted (ascending or descending). 

Bonus points: filters for organization name

**Comment by mieubrisse** [comment]
@leeederek yep ACK that it's not good; we're actually thinking of sorting by stars. this might come in September, but possibly October (shouldn't be longer than that)
